### PR TITLE
.NET 6 upgrade

### PR DESCRIPTION
**Pull request by helto4real**
Yet another booring .NET 6 upgrade and fixes all warnings PR... Upgrading .NET is an dark and lonely business but someone gotta do it!

### CODE REVIEW

File: .github/workflows/ci_build.yml [hunk 1]
```diff
@@ -14,12 +14,12 @@ jobs:
...
     steps:
-      - name: 📤 Checkout the repository 
+      - name: 📤 Checkout the repository
         uses: actions/checkout@main
-      - name: 🥅 Install .Net 5
+      - name: 🥅 Install .Net 6
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: '5.0.100' # SDK Version
+          dotnet-version: "6.0.100" # SDK Version
```
Comment (BeeHiveJava) [inline .github/workflows/ci_build.yml]: 6.0.x?

File: .github/workflows/push_nuget.yml [hunk 2]
```diff
@@ -22,11 +22,11 @@ jobs:
           latest=$(git describe --tags $(git rev-list --tags --max-count=1))
           echo Current version:    $latest
           echo "::set-output name=version::$latest"
-      - name: 🥅 Install .Net 5
+      - name: 🥅 Install .Net 6
         uses: actions/setup-dotnet@v1
         with:
-            dotnet-version: '5.0.100' # SDK Version
+          dotnet-version: "6.0.100" # SDK Version
```
Comment (BeeHiveJava) [inline .github/workflows/push_nuget.yml]: 6.0.x?

**Review by BeeHiveJava — APPROVED** [review]

File: .github/workflows/ci_build.yml (hunk 1 above)
Comment (helto4real) [reply .github/workflows/ci_build.yml]: yes!

File: .github/workflows/push_nuget.yml (hunk 2 above)
Comment (helto4real) [reply .github/workflows/push_nuget.yml]: Yes thanks. Will fix!

**Pull request merged**